### DISCUSSION TITLE
Feature Req: Add Schedule info to Slack alert message

### DISCUSSION THREAD

**Discussion by cbini** [dagster-io/dagster#7768] [Ideas]
I have a Job that I create several Schedules from, all with different names and config values. It would be great if you could add at least the name of the Schedule run to the Slack alert, so I can more easily disambiguate which run of the Job failed. 

Presently, the message only contains the name of the Job, which is the same for all of the schedules:
![image](https://user-images.githubusercontent.com/5003326/167176119-4da8f104-edae-48eb-a58b-7f4ae9b239be.png)

**Comment by cbini** [comment]
Relatedly, I also recommend making the link to the run more clear. Perhaps include the Run ID as the hyperlink instead of the Job name.

**Comment by rexledesma** [comment]
This is more of a feature request, so converting this into an issue: https://github.com/dagster-io/dagster-cloud/issues/3!
